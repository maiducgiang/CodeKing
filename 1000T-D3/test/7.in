2
2418 2484 8903 1027
2651 2854 9905 6349
