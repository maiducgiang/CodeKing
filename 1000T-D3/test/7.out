3070884
18841914
